### PR TITLE
Fixing get_pixel_locations()

### PR DESCRIPTION
**Pull request by altuson**
Updated get_pixel_locations():

- Switched order of col, row. 
- Added one to pixel index.
- Removed rounding of col, row.

### CODE REVIEW

File: src/tesswcs/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 PACKAGEDIR = os.path.abspath(os.path.dirname(__file__))
 
-version = "1.1.2"
+version = "1.1.3"
```
Comment (christinahedges) [inline src/tesswcs/__init__.py:6]:
```suggestion
__version__ = "1.1.3"
```

**Review by christinahedges — COMMENTED** [review]
Hi @altuson I left a suggestion to fix another bug that `version` should be `__version__` in the code base. 

I see you have a test here, how do you arrive at the "true" value for the test? (i.e. how do you know that the test answer is correct?

**Comment by altuson** [comment]
Hi @christinahedges, I made your commit and also added comments to the test. The "true" value is determined by independently querying tessrip to get the average WCS for the sector/camera/ccd, then using the function wcs_world2pix() with origin=1 to convert the coordinate to col, row.

**Comment by christinahedges** [comment]
Hi @altuson this looks great to me. I've updated this to ensure the tests pass and the `black` and `flake8` continuous integrations run.

**Pull request merged**